sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/Statsbeat.cs

```diff
@@ -101,6 +97,35 @@ internal static class Statsbeat
             "westus3",
         };
 
+        internal Statsbeat(ConnectionVars? connectionStringVars)
```

Comment:
ConnectionVars shouldn't be null :)

```suggestion
        internal Statsbeat(ConnectionVars connectionStringVars)
```